### PR TITLE
feat(checkout): CHECKOUT-7125 Adding additional aria attributes to coupon code and gift certificate buttons on the cart page

### PR DESCRIPTION
**Pull request by bc-NikitaPuzanenko**
#### What?

Adding aria-hidden attributes to coupon code and gift certificate containers on the cart page. Also adding aria labels to coupon code and gift certificate apply buttons on the cart page.

#### Requirements

- [x] CHANGELOG.md entry added (required for code changes only)

#### Screenshots (if appropriate)

After changes:
Screenshot 2023-08-23 at 4 42 50 PM

### CODE REVIEW

**Review by BC-krasnoshapka — APPROVED** [review]

File: templates/components/cart/gift-certificate-input.html [hunk 1]
```diff
@@ -5,11 +5,11 @@
     <button class="gift-certificate-cancel" style="display: none;">{{lang 'cart.coupons.cancel'}}</button>
 </div>
 
-<div class="cart-form gift-certificate-code" style="display: none;">
+<div class="cart-form gift-certificate-code" aria-hidden="true" style="display: none;">
     <form class="form form--hiddenLabels cart-gift-certificate-form" method="post" action="{{urls.cart}}">
         <label class="form-label" for="certcode">{{lang 'cart.gift_certificates.cert_code'}}</label>
         <input class="form-input" id="certcode" type="text" name="certcode" value="" placeholder="{{lang 'cart.gift_certificates.add_cert_code'}}">
-        <input class="button button--primary button--small" type="submit" value="{{lang 'cart.coupons.button'}}">
+        <input class="button button--primary button--small" aria-label="Apply Gift Certificate" type="submit" value="{{lang 'cart.coupons.button'}}">
```
Comment (yurytut1993) [inline templates/components/cart/gift-certificate-input.html:12]: Should this be added to translation file?

File: templates/components/cart/coupon-input.html [hunk 2]
```diff
@@ -5,11 +5,11 @@
     <button class="coupon-code-cancel" style="display: none;">{{lang 'cart.coupons.cancel'}}</button>
 </div>
 
-<div class="cart-form coupon-code" style="display: none;">
+<div class="cart-form coupon-code" aria-hidden="true" style="display: none;">
     <form class="form form--hiddenLabels coupon-form" method="post" action="{{urls.cart}}">
         <label class="form-label" for="couponcode">{{lang 'cart.coupons.coupon_code'}}</label>
         <input class="form-input" data-error="{{lang 'cart.coupons.empty_error'}}"  id="couponcode" type="text" name="couponcode" value="" placeholder="{{lang 'cart.coupons.coupon_code'}}">
-        <input class="button button--primary button--small" type="submit" value="{{lang 'cart.coupons.button'}}">
+        <input class="button button--primary button--small" type="submit" aria-label="Apply Coupon" value="{{lang 'cart.coupons.button'}}">
```
Comment (yurytut1993) [inline templates/components/cart/coupon-input.html:12]: this should be also added to `en.json` file as you've done it for gift certificate

**Review by yurytut1993 — APPROVED** [review]
LGTM 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Display fees on cart page [#2376](https://github.com/bigcommerce/cornerstone/pull/2376)
 - Replace Twitter logo with X logo within social sharing and social link components [#2387](https://github.com/bigcommerce/cornerstone/pull/2387)
 - Added nvm config [#2389](https://github.com/bigcommerce/cornerstone/pull/2389)
-- Displaying the Hidden cart_order_source Input Field on PDP page [#2392](https://github.com/bigcommerce/cornerstone/pull/2392)
```
Comment (BC-krasnoshapka) [inline CHANGELOG.md:11]: @bc-NikitaPuzanenko just noticed this line was suddenly removed

Comment (bc-NikitaPuzanenko) [reply CHANGELOG.md:11]: Sorry about that not sure what happened.

**Comment by mattcoy-arcticleaf** [comment]
aria-hidden should not be added to elements that are displayed none;

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden

**Comment by mattcoy-arcticleaf** [comment]
@bc-yevhenii-buliuk , the submit input tag can be changed to a button tag and then you can place 1 or more spans inside.

**Comment by bc-yevhenii-buliuk** [comment]
@mattcoy-arcticleaf yes, we can change the submit input to the button tag and add a span inside, but it will change the UI, the content inside the span will stretch the button width, which is not intended by the design.
For example, now the submit input is called
`| Apply |`
and by changing it to a button and adding a span inside, it can look like
`| Apply gift certificate |`

**Pull request merged**